### ISSUE TITLE
Add 'dictionaries' metadata to all tag pages

### ISSUE THREAD

**Issue by hegemonic** [jsdoc3/jsdoc3.github.com#81]
Each tag page should have a `dictionaries` metadata item. The `dictionaries` metadata should be an array of strings indicating which JSDoc tag dictionaries support each tag (currently `jsdoc`, `closure`, or both).

Once this change is complete, we should file an issue to update the template so it displays which dictionaries include each tag.

**Comment by hegemonic** [comment]
Duplicate of #157.
